test/simulator/test/org/apache/cassandra/simulator/test/ShortAccordSimulationTest.java

```diff
@@ -24,6 +24,74 @@
 
 import org.apache.cassandra.simulator.paxos.AccordSimulationRunner;
 
+/**
```

Comment:
this was unrelated.  I found a failing seed so wanted to run it, and i find this documentation critical to run the tests... so added it